### PR TITLE
Add fit/fitTo, clean up Hershey, support functors in and, cut, etc.

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: api/shape/Hershey.js [hunk 1]
```diff
@@ -1431,10 +1431,8 @@ export const toPaths = (letters) => {
     .outline();
 };
 
-export const ofSize = (size) => (text) => toPaths(text).scale(size);
+export const ofSize = (text, size) => toPaths(text).scale(size);
```
Comment (BarbourSmith) [inline api/shape/Hershey.js:1434]: Does this change how Hershey works? How would I use it after the change?

File: api/shape/main.js [hunk 2]
```diff
@@ -280,7 +281,9 @@ export { minkowskiSum } from './minkowskiSum.js';
 export { move } from './move.js';
 export { noVoid } from './noVoid.js';
 export { notAs } from './notAs.js';
+export { nth } from './nth.js';
```
Comment (BarbourSmith) [inline api/shape/main.js:284]: It's fun to see these get added 👍 👍

**Review by BarbourSmith — APPROVED** [review]
Looks good to me

File: api/shape/Hershey.js (hunk 1 above)
Comment (pentacular) [reply api/shape/Hershey.js:1434]: Ah, it changes it to match how Text() works.

Hershey(12)('Hello') becomes Hershey('Hello', 12)

**Pull request merged**